rllib/algorithms/apex_dqn/tests/test_apex_dqn.py

```diff
@@ -15,7 +16,7 @@
 
 class TestApexDQN(unittest.TestCase):
     def setUp(self):
-        ray.init(num_cpus=4)
+        ray.init()
```

Comment:
Not sure why we change this here?

Reply:
If you specify the num_cpu in ray init it won't run on the cluster. So why specify it if ray can automatically assign itself the max number of resources available?